src/containers/exchange/components/TradeBook.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Pblock } from "components/PageLayout";
+import { ErrorPanel } from "components/MsgPanels";
+import { CustomTable } from "components/Table";
+
+export default class TradeBook extends React.Component {
+    render() {
+        const { header, userAccountAddress } = this.props;
+        const { trades, error, isLoading } = this.props.trades;
+        const dataKeys = ['blockTimeStampText', 'type', 'direction', 'price', 'tokenValue', 'ethAmountRounded', 'tokenAmount'];
+        const unit = ['', '', '', 'A-EUR/ETH', 'A€', 'ETH', 'A€'];
+        const headerData = {
+          'blockTimeStampText': 'Date',
+          'type': 'Type',
+          'direction': 'Direction',
+          'price': 'Price',
+          'tokenValue': 'Token Value',
+          'ethAmountRounded' : 'Eth Amount',
+          'tokenAmount': 'Token Amount'
+        }
+
+        return (
+            <Pblock loading={isLoading} header={header}>
```

Comment:
this loading doesn't work, I can only see a Connecting... message and table headers shown.
 Also a msg insteasd of emtpy table with table headers:
https://trello.com/c/PztAuJe7/107-my-trade-history-you-have-no-trade-history-instead-of-empty-tables